packages/terra-button/src/Button.module.scss

```diff
@@ -33,10 +33,10 @@
     font-size: var(--terra-button-font-size, 1rem);
     line-height: var(--terra-button-line-height, 1.429);
     outline: none;
-    padding-bottom: var(--terra-button-vertical-padding, 0.286rem);
+    padding-bottom: var(--terra-button-vertical-padding-bottom, 0.286rem);
     padding-left: 0;
     padding-right: 0;
-    padding-top: var(--terra-button-vertical-padding, 0.286rem);
+    padding-top: var(--terra-button-vertical-padding-top, 0.286rem);
```

Comment:
I think we can drop the word `vertical` from these variable names now that they are explicitly top and bottom.

Reply:
removed vertical: https://github.com/cerner/terra-core/pull/2125/commits/a3506cef10701a7f7581ac4212d9294c04917a28#diff-6183fae6ac908ad4f76e328f50a3aeaeR36